modules/client/src/client-common/schemas/common.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  InvalidAddressError,
+  InvalidBigintError,
+  InvalidBigNumberishError,
+  InvalidDataError,
+  InvalidDateError,
+  InvalidIpfsUriError,
+  InvalidPermissionError,
+} from "@aragon/sdk-common";
+import { isAddress } from "@ethersproject/address";
+import { mixed, object, string } from "yup";
+import { Permissions } from "../../interfaces";
+import { isBigNumberish } from "@ethersproject/bignumber/lib/bignumber";
+import { isIpfsUri } from "../utils";
+
+export const addressSchema = string().notRequired().test(
+  "isAddress",
+  new InvalidAddressError().message,
+  (value) => value ? isAddress(value) : true,
+);
+
+export const ipfsUriSchema = string().notRequired().test(
+  "isIpfsUri",
+  new InvalidIpfsUriError().message,
+  (value) => value ? isIpfsUri(value) : true,
+);
+
+// address list or ens
+export const addressOrEnsSchema = string().test(
+  "isAddressOrEns",
+  new InvalidAddressError().message,
+  // check ens
+  (value) => value ? isAddress(value) : true,
+);
+// permissions
+export const permissionSchema = string().test(
+  "isPermission",
+  new InvalidPermissionError().message,
+  (value) => value ? Object.keys(Permissions).includes(value) : true,
+);
+
+// uint8array
+export const uint8ArraySchema = mixed().test(
+  "isUint8Array",
+  new InvalidDataError().message,
+  (value) => value ? value instanceof Uint8Array : true,
+);
+
+// uint8array
+export const bigNumberishSchema = mixed().test(
+  "isBigNumberish",
+  new InvalidBigNumberishError().message,
+  (value) => value ? isBigNumberish(value) : true,
```

Comment:
I added this as a temporary measure until we decide the new `proposalId` format

the logic with the `value? something : true` is that we don't always know if the value is defined, so if it is defined we do the validations, if it is not, we don't do them because if we do it we can't have optional values.

If a required field receives an `undefined`, will pass this, but it won't pass the `required()` flag